src/Materials/effect.ts

```diff
@@ -336,6 +352,9 @@ export class Effect implements IDisposable {
             }, this._engine);
         });
         this._loadShader(fragmentSource, "Fragment", "Pixel", (fragmentCode) => {
+            if (processorOptions.processor?.removeCommentsBeforeProcessing) {

```

Comment:
Why adding this here and not in the processor ?

Reply:
Code moved to `ShaderProcessor`